apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -222,11 +222,20 @@ public void disableSubscriptions() {
 
   /**
    * @return The default {@link CacheHeaders} which this instance of {@link ApolloClient} was configured.
+   * @deprecated Use getDefaultCacheHeaders() instead
    */
+  @Deprecated
```

Comment:
We had other instances where we had `get`. Plus, it is nicer in Kotlin. That's why for consistency I wanted to deprecate this one.